src/data/actions/assets.js

```diff
@@ -6,9 +6,9 @@ export const requestAssetsSuccess = response => ({
   data: response.assets,
 });
 
-export const getAssets = assetsParameters =>
+export const getAssets = (assetsParameters, courseDetails) =>
   dispatch =>
-    clientApi.requestAssets(assetsParameters.courseId, {
+    clientApi.requestAssets(courseDetails.id, {
```

Comment:
@dsjen updated the API calls to use the ID from courseDetails instead of the hard coded value :)

Reply:
w00t!

Reply:
w00t!